Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "vart"
 publish = true
-version = "0.1.1"
+version = "0.1.2"
```

Comment:
Would still recommend version upgrades to be separate PRs :P But it's fine for now obvs - it can stay in

Reply:
Yes you are right in this

Reply:
Will do this from next time